### DISCUSSION TITLE
〔求助〕求助！绳网怎么发图片！

### DISCUSSION THREAD

**Discussion by lbghhj** [share121/inter-knot#412] [其他]
安比对我做奇怪动作，想展示并求助，没想到发不出图片！求教发图片

**Comment by Kamisato100** [comment]
你GitHub输入框右上角有个回形针的图标，鼠标移上去会显示“Attach files”，点击之后选择你想上传的图片，然后等一会等上传成功变成完整的链接就可以发出来了，大家记得不要发不和谐的图片哦！
